android/app/build.gradle

```diff
@@ -104,7 +102,7 @@ android {
 
     namespace 'io.mosip.residentapp'
 
-    
+
     namespace 'io.mosip.residentapp'
```

Comment:
Delete this duplicate entry for namespace

Reply:
removed.